stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/avatar/AvatarView.kt

```diff
@@ -69,10 +112,18 @@ public class AvatarView : AppCompatImageView {
     }
 
     override fun onDraw(canvas: Canvas) {
-        if (drawable != null) {
-            super.onDraw(canvas)
-            drawBorder(canvas)
-            drawOnlineStatus(canvas)
+        when (avatarStyle.avatarType) {
+            AvatarType.PICTURE -> {
+                if (drawable != null) {
+                    super.onDraw(canvas)
+                    drawBorder(canvas)
+                    drawOnlineStatus(canvas)
+                }
+            }
+            AvatarType.COLOR -> {
+                drawColor(canvas, avatarStyle)
+                drawInitials(canvas, avatarInitials)
+            }
```

Comment:
I am just wondering if we don't need to draw the border and online status for this type.

Reply:
ops, we do. My mistake